# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: da26bac32c32
# Full Hash: da26bac32c327cd814c4b13630061f7a5c07c339
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 3
# Description:
#   Bug 1654112 - Clean up and sort Audio/Video/MediaConduit includes. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102289
# ==============================================================================

diff -r ceed49af0fad -r da26bac32c32 dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Mon Jan 18 17:32:49 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Tue Jan 19 10:59:28 2021 +0100
@@ -5,27 +5,22 @@
 #ifndef MEDIA_CONDUIT_ABSTRACTION_
 #define MEDIA_CONDUIT_ABSTRACTION_
 
-#include "nsISupportsImpl.h"
-#include "nsXPCOM.h"
-#include "nsDOMNavigationTiming.h"
-#include "mozilla/RefPtr.h"
+#include <vector>
+#include <set>
+
+#include "CodecConfig.h"
+#include "ImageContainer.h"
+#include "jsapi/PeerConnectionCtx.h"
+#include "jsapi/RTCStatsReport.h"
+#include "MediaConduitErrors.h"
 #include "mozilla/RefCounted.h"
-#include "mozilla/UniquePtr.h"
-#include "CodecConfig.h"
+#include "TaskQueueWrapper.h"
 #include "VideoTypes.h"
-#include "jsapi/PeerConnectionCtx.h"
-#include "MediaConduitErrors.h"
-#include "jsapi/RTCStatsReport.h"
-#include "TaskQueueWrapper.h"
 
-#include "ImageContainer.h"
-
+// libwebrtc includes
 #include "api/video/video_frame_buffer.h"
 #include "call/call.h"
 
-#include <vector>
-#include <set>
-
 namespace webrtc {
 class VideoFrame;
 }